src/dist/replication/meta_server/meta_bulk_load_service.h

```diff
@@ -306,11 +306,14 @@ class bulk_load_service
         return (_bulk_load_app_id.find(app_id) != _bulk_load_app_id.end());
     }
 
+    mss::meta_storage *get_sync_bulk_load_storage() const { return _sync_bulk_load_storage.get(); }
+
```

Comment:
return `mss::meta_storage &` is a little better

Reply:
`get_sync_bulk_load_storage` is a redundant function, I remove it.